src/vishwasCDec19/assignment3/Q4SumOfNo.java

```diff
@@ -0,0 +1,30 @@
+package vishwasCDec19.assignment3;
+/*
+ *  WAP to find sum of user given number
+ */
+import java.util.Scanner;
+public class Q4SumOfNo 
+{
+	static int total;
+
+	static void totalAdd(String number)
+	{
+
+		for (int i = 0; i < number.length(); i++)
+		{
+			Character u;
+			u = number.charAt(i);
+			String c = u.toString();
+			int x = Integer.parseInt(c);
+			total = total + x;
```

Comment:
better implementation is below:

total = total + Integer.parseInt(String.valueOf(str.charAt(i)));

Just avoid extra variable and please give variable name logical.